zulipterminal/ui_tools/boxes.py

```diff
@@ -490,6 +490,16 @@ def autocomplete_emojis(self, text: str, prefix_string: str
 
         return emoji_typeahead, emojis
 
+    def footer_notify_invalid_recipient(self, error_msg: str) -> None:
+        footer_message = [
+            error_msg + ' Use ',
+            ('code', primary_key_for_command('AUTOCOMPLETE')),
+            ' or ',
+            ('code', primary_key_for_command('AUTOCOMPLETE_REVERSE')),
+            ' to autocomplete.'
+        ]
+        self.view.set_footer_text(footer_message, 3)
```

Comment:
The method is calling an existing method, `set_footer_text`, with a particular error message. This could lead us to a state where we have multiple methods that essentially do the same thing with different arguments for different _error_ types.

Another way could be to just have an `ERROR_TEMPLATES` data structure that could be used to pass an appropriate error message to the existing `set_footer_text` method.

Reply:
Would it make sense to define the `ERROR_TEMPLATES` in a separate module?